modules/ols-attaching-a-resource-object-to-your-query.adoc

```diff
@@ -24,6 +24,10 @@ You can attach the following resources:
 * `CronJob`, `DaemonSet`, `Deployment`, `Job`, `Pod`, `ReplicaSet` and `StatefulSet` from *Workloads* in the {ocp-product-title} web console.
 
 * `Alert` from *Observe* in the {ocp-product-title} web console.
+
+* `YAML` from *Networking* in the {ocp-product-title} web console.
+
+* `Services`, `Routes`, `Ingresses`, `NetworkPolicies` from *Networking* in the {ocp-product-title} web console.
```

Comment:
Just curious, since I don't know the context of this doc - is there a reason that the YAML option is on its own list item when it's technically from Networking as well? Not suggesting a fix, just wondering if that's intentional.

Reply:
great catch! that was an early draft that I meant to delete!